01week/datatypes.js

```diff
@@ -0,0 +1,87 @@
+'use strict'
+
+
+//1. display the time and the dat
+const today =  new Date();
+console.log(today);
+
+const day = today.getDay();
+console.log(day);
+
+const dayOfTheWeek = ['Sunday','Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday'];
+console.log("It is : " + dayOfWeek[day]);
+
+
+const hour = today.getHours();
+console.log('hour ' + hour);
+
+const minute = today.getMinutes();
+console.log('minutes ' + minute);
+
+const  second = today.getSeconds();
+console.log('seconds ' + second);
+
+
+ // 2. returning the type of data type function
+  const ReturnTypeOf(input)=>{
+    return typeof(input);
+ }
+ console.log(typeof 42);
+// NUMBER DATA TYPE
+
+console.log(typeof 'Javascript is difficult');
+// STRING DATA TYPE
+
+console.log(typeof false);
+// BOOLEAN DATA TYPE
+
+
+ // 3. turning a number to into a string 
+  const number = 10000;
+  const numberToString= number.ToString;
+  console.log(number);
+ 
+ // 4. return the sum of two numbers
+  const addition=(number1, number2) => {
+   return number1 + number2;
+  }
+  addition(5,5);
+ // 5.  return a string to number 
+   const stringToNumber=(string){
+     return Number(string);
+   }
+   stringToNumber('900');
+ 
+ 
+ // 6. two truths
+ 
+  const doubleTrue=()=>{
```

Comment:
This function evaluates if the boolean true evaluates to true. I need a function that takes in two parameters and if they both evaluate to true, run a block of code.